fedimint-bin-tests/src/federation.rs

```diff
@@ -45,7 +45,7 @@ fn fedimint_env(peer_id: usize) -> anyhow::Result<HashMap<String, String>> {
 pub struct Federation {
     // client is only for internal use, use cli commands instead
     client: Arc<UserClient>,
-    members: Vec<Fedimintd>,
+    members: Vec<Option<Arc<Fedimintd>>>,
```

Comment:
no? I meant use ProcessHandle's internal arc in ProcessHandle::kill.

Sorry, if I wasn't clear.